### ISSUE TITLE
Can redis component in gin-cache be upgraded to latest version 8.11.5?

### ISSUE THREAD

**Issue by rts-gordon** [chenyahui/gin-cache#30]
Hi @chenyahui 

   gin-cache is very useful, thanks of your work.
   Can redis component in gin-cache be upgrade to latest version 9(v9.0.2)?
  
  thank you.

**Comment by chenyahui** [comment]
https://github.com/chenyahui/gin-cache-redis

**Comment by rts-gordon** [comment]
Sorry, I mean is: Can redis in gin-cache be upgraded to version 9.0.2?  Not gin-cache-redis.

**Comment by chenyahui** [comment]
you can use
```
import (
	cache "github.com/chenyahui/gin-cache"
	redisbackend "github.com/chenyahui/gin-cache-redis"
	"github.com/gin-gonic/gin"
	"github.com/go-redis/redis/v9"
	"time"
)

func main() {
	app := gin.New()

	redisStore := redisbackend.NewRedisV9Store(redis.NewClient(&redis.Options{
		Network: "tcp",
		Addr:    "127.0.0.1:6379",
	}))

	app.GET("/hello",
		cache.CacheByRequestURI(redisStore, 2*time.Second),
		func(c *gin.Context) {
			c.String(200, "hello world")
		},
	)
	if err := app.Run(":8080"); err != nil {
		panic(err)
	}
}
```

**Comment by chenyahui** [comment]
The redis version of gin-cache will not upgrade to v9, because it's a break change.

**Comment by rts-gordon** [comment]
Got it, thank you any way.

**Comment by rts-gordon** [comment]
The go-redis version 9 is for Redis version 7,  go-redis version 8 is for Redis version 6. 

Can  go-redis be upgraded to version 8.11.5? There are a lot of bugs have been fixed from version 8.11.4.
